src/CLI/CLI.csproj

```diff
@@ -55,13 +55,22 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>DisplayStrings.resx</DependentUpon>
     </Compile>
+    <Compile Update="Resources\OptionsHelpText.Designer.cs">
```

Comment:
I put these into a separate resource file for 2 reasons:
1. There was already a logical grouping that I wanted to preserve
2. These strings need to be public for CommandLineParser to use them. The existing strings can be internal.